### ISSUE TITLE
PostgreSQL: query with REPLACE produced incorrect Go code

### ISSUE THREAD

**Issue by zer0main** [kyleconroy/sqlc#559]
Query:
```
UPDATE <table_name> SET <field> = REPLACE(<field>, $1, $2);
```
Produced the following args struct:
```
type ReplaceParams struct {
        Replace   interface{}
        Replace   interface{}
        Replace_3 interface{}
        Replace_3 interface{}
}
```
Obviously, Go compiler complains about the redeclared field.

**Comment by zer0main** [comment]
Any updates on this issue?

**Comment by kyleconroy** [comment]
Hey, sorry this took so long to get fixed. This was fixed in #636 and is available in the latest tip release, which means it will be in the v1.5.0 release.
